crates/uv/tests/snapshots/ecosystem__warehouse-lock-file.snap

```diff
@@ -5,9 +5,8 @@ expression: lock
 version = 1
 requires-python = ">=3.11, <3.12"
 environment-markers = [
-    "python_version < '3.11'",
-    "python_version >= '3.11.[X]' and python_version < '3.12'",
-    "python_version >= '3.12' and python_version < '3.13'",
+    "python_version < '3.12'",
+    "python_version < '3.13'",
     "python_version >= '3.13'",
```

Comment:
shouldn't this remove too? since it doesn't fill in `>=3.11, <3.12` required range.

Reply:
We don't respect upper-bounds on `requires-python`.